scripts/build_and_test.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# build
+docker-compose -f docker-compose.yml build static_site
+
+# test
+docker-compose run -u $(id -u ${USER}):$(id -g ${USER}) --publish 4000:4000 --rm --entrypoint "bundle exec jekyll serve -H 0.0.0.0" -d static_site
+sleep 20
+curl localhost:4000 | grep "Join the Google Group"
```

Comment:
I'm concerned that since we are running this in a separate shell script, only the end result of the script is returned back to Travis when reporting status.  So, if any step fails, the script just goes on to the next step... and as long as `docker compose down` returns an error code of 0 (which it will 99.99% of the time), then the status in Travis will _always_ be green (even if a failure occurs).  
Therefore, we need to [fail fast and be aware of exit codes](https://gist.github.com/leolorenzoluis/0aad69719267536d0b7a79946edbfcb7#file-bash-best-practices-L13).
Before the build step, can we add `set -eo pipefail` to the script?
An example of where we do this elsewhere is [here](https://github.com/CMSgov/bcda-app/blob/master/unit_test.sh#L6-L7).